v2/tools/asoctl/internal/importing/import_skipped_error.go

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package importing
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// ImportSkippedError is an error that indicates that a resource cannot be imported for a reason we know about.
+// This allows us to continue the import even if some expected errors occur.
+type ImportSkippedError struct {
```

Comment:
Are these errors all by definition "known and thus ignorable"? Or do we need a `Severity` or something like that attached to this too?

Reply:
Put another way, is there any such thing as a "Known and _not_ ignorable" error?

Reply:
Any not-ignorable error can just be returned and will do the right thing. We only need a special case for errors that are non-fatal.